DependencyInjection/CmfRoutingExtension.php

```diff
@@ -82,7 +87,20 @@ private function setupDynamicRouter(array $config, ContainerBuilder $container,
         $loader->load('routing-dynamic.xml');
 
         // strip whitespace (XML support)
-        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class', 'route_filters_by_id') as $option) {
+        foreach (array('controllers_by_type', 'controllers_by_class', 'templates_by_class') as $option) {
+            $config[$option] = array_map(function ($items) {
+                foreach ($items as $key => $item) {
+                    if (is_array($item)) {
+                        $items[$key]['value'] = trim($item['value']);
+                    } else {
+                        $items[$key] = trim($item);
```

Comment:
There also is an WhiteListTest. I think is a point of the controller resolver, that tries to get the controller as a service, isn't it?